### ISSUE TITLE
doesn't work when input is an object

### ISSUE THREAD

**Issue by nicolodavis** [darionco/rollup-plugin-web-worker-loader#7]
Rollup allows you to specify inputs as [objects](https://rollupjs.org/guide/en/) (or arrays of strings):

```js
export default {
  input: {
    'a': './src/a.js',
    'b': './src/b.js',
  }
  ...
}
```

This plugin [expects](https://github.com/darionco/rollup-plugin-web-worker-loader/blob/master/src/index.js#L36) `input` to be a string.

**Comment by darionco** [comment]
good catch, I use the input to compute the paths for the source maps... the problem is bigger than just supporting the object syntax but supporting multiple input files.

I'll take a deeper look, sounds like an interesting problem!

**Comment by darionco** [comment]
a fix was published in v0.8.1
